tests/test_computetools.py

```diff
@@ -37,9 +37,9 @@ def test_push():
     assert np.allclose(x_i, xfinal, rtol=1e-4)
     x_i = np.array([[0, 0, 0.0]], dtype=np.float)
     p_i = 0 * v_i
-    E = np.array([[0, 0, 1e100]], dtype=np.float)
-    B = np.array([[0, 0, 1e100]], dtype=np.float)
+    E = np.array([[0, 0, 1e10]], dtype=np.float)
+    B = np.array([[0, 0, 1e10]], dtype=np.float)
```

Comment:
I meant 10, not 1e10.
```suggestion
    E = np.array([[0, 0, 10]], dtype=np.float)
    B = np.array([[0, 0, 10]], dtype=np.float)
```